content/_code-samples/checks/py/cash_check.py

```diff
@@ -0,0 +1,75 @@
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.utils import xrp_to_drops
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+from xrpl.clients import JsonRpcClient
+from xrpl.models import CheckCreate, IssuedCurrencyAmount
+
+# create a client
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234")
+
+
+# cash an xrp check
+
+# check id
+check_id = str
```

Comment:
similar to comment on create_check, if you could create a check_id and make sure it runs that would be great

Reply:
All good